website/content/docs/commands/web.mdx

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs
+page_title: Web REPL - Command
+description: |-
+  The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
+
+---
+
+
+# Web REPL
+
+The Vault web user interface (UI) includes an advanced mode that mimics some basic commands from the Vault CLI. It can be useful a user is more familiar with the CLI or API as it lets a user directly input the paths they wish to manipulate.
+
+~> **Note:** The Vault Web REPL is _not_ a full terminal emulator. Features like environment varialbes, HEREDOC, or piping of data or files will not work unless explicitly documented.
+
+## Command history
+The Web REPL will keep a history of all of the commands you've entered in your current session. If you refresh the browser or use the `clearall` command, this history will be reset.
+
+To cycle through the command history, you can use the up and down arrows when the REPL input has focus. If you reach either end of the history, it will cycle back to the the other side and continue looping through them.
```

Comment:
Only the Up ⬆️ arrow seems to loop 🤔 Down ⬇️  stops when it reaches the end

Reply:
Oh whoops, I was mashing keys too quickly and didn't catch this. I'll update it!

Reply:
Updated!